audio-worklet/src/bitcrusher-demo.js

```diff
@@ -17,16 +17,15 @@ class BitcrusherDemo {
   /**
    * Initalizes and modifies bitcrusher settings in response to GUI input.
    * @param {AudioContext} context the Audio Context
-   * @param {String} containerId container for GUI elements
-   * @param {Boolean} useWorkletAndScriptProcessor switch depending on browser
+   * @param {Boolean} useWorklet switch depending on browser
```

Comment:
Nit:  `useWorklet`  implies to me that you're only using a worklet.  But in fact it's more like `enableWorket` so that you can use a worklet but you don't have to.  Which is the case in this demo.

The comments describe the purpose of this clearly, but having a name that reflects that is better.